com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -258,58 +321,122 @@ def get_view_items(self):
+
+    The object contains a JSON representation of the resource, whose properties can be accessed directly using dot
+    notation.
     """
     pass
 
 
 class Host(_ResourceElement):
     """The host element resource provides access to information about a host that is allocated to a domain as a
     resource for running Streams services and applications.
+
+    The object contains a JSON representation of the resource, whose properties can be accessed directly using dot
+    notation.
     """
     pass
 
 
 class Job(_ResourceElement):
     """The job element resource provides access to information about a submitted job within a specified instance.
+
+    The object contains a JSON representation of the resource, whose properties can be accessed directly using dot
+    notation.
+
+    Example:
+        >>> from streamsx import rest
+        >>> sc = rest.StreamingAnalyticsConnection()
+        >>> instances = sc.get_instances()
+        >>> jobs = instances[0].get_jobs()
+        >>> print (jobs[0].health)
+        healthy
     """
     def get_views(self, name=None):
+        """Get the list of views associated with this job resource.
+
+        Args:
+            name(str, optional): only return views containing property **name** that matches `name`.  `name` can be a
+                regular expression.  If `name` is not supplied, then all view resources are returned.
+
+        Returns:
+            :py:obj:`list` of :py:class:`View`: list of view resources matching `name`.
+        """
         return self._get_elements(self.views, 'views', View, name=name)
 
     def get_domain(self):
+        """Get the Streams domain that owns this job element resource
+
+        Returns:
+            :py:class:`Domain`: Domain owning this job element resource
+        """
         return Domain(self.rest_client.make_request(self.domain), self.rest_client)
 
     def get_instance(self):
+        """Get the Streams instance that owns this job element resource
+
+        Returns:
+            :py:class:`Instance`: Instance owning this job element resource
+        """
         return Instance(self.rest_client.make_request(self.instance), self.rest_client)
 
     def get_hosts(self):
+        """Get the list of Host element associated with this job resource
+
+        Returns:
+            :py:obj:`list` of :py:class:`Host`: list of Host resource objects.
+        """
         return self._get_elements(self.hosts, 'hosts', Host)
 
     def get_operator_connections(self):
+        """Get the list of OperatorConnection element associated with this job resource
+
+        Returns:
+            :py:obj:`list` of :py:class:`OperatorConnection`: list of OperatorConnection resource objects.
+        """
         return self._get_elements(self.operatorConnections, 'operatorConnections', OperatorConnection)
 
     def get_operators(self):
+        """Get the list of Operator element associated with this job resource
```

Comment:
Same comment as resource but for element, at this point it's a list of operators.